server/core/src/test/java/org/infinispan/server/core/AbstractProtocolServerTest.java

```diff
@@ -50,7 +52,7 @@ public void testStartingWithoutTransport() {
       b.startTransport(false);
       AbstractProtocolServer server = new MockProtocolServer();
       try {
-         server.start(b.build(), new DefaultCacheManager());
+         server.start(b.build(), new DefaultCacheManager(new ConfigurationBuilder().build()));
```

Comment:
No, but the default constructor no longer provides a default cache, unless you pass it in.
The default constructor alone will give you a cache manager with no caches. You can use defineConfiguration() to provide some